bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/ScriptExecution.java

```diff
@@ -91,6 +91,10 @@ public static Timer createTimer(AbstractInstant instant, Procedure0 closure) {
             dataMap.put("procedure", closure);
             dataMap.put("timer", timer);
             JobDetail job = newJob(TimerExecutionJob.class).withIdentity(jobKey).usingJobData(dataMap).build();
+            if (TimerImpl.scheduler.checkExists(job.getKey())) {
+                TimerImpl.scheduler.deleteJob(job.getKey());
```

Comment:
For example:
This code (implemented in ScriptEngineOSGiTest)
```groovy
        def instant = new DateTime(2015, 12, 21, 9, 48)
        def closure = new Runnable() {public void run() {}}
        JobKey jobKey = new JobKey(instant.toString() + ": " + closure.toString())
        println  jobKey.toString()
```
lets to this String
```
DEFAULT.2015-12-21T09:48:00.000+01:00: org.eclipse.smarthome.model.script.tests.scriptengine.ScriptEngineOSGiTest$1@3e5499cc
```